frontend/src/components/Header/Header.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import '../Button/button.css';
+import './header.css';
+import { Link } from 'react-router-dom';
+
+export default class Header extends Component {
+
+    render() {
+        const  { divCabecalho, divTitulo, divBtn } = this.props
+        return (
+                <div className={divCabecalho}>
+                    <div className={divTitulo}>
+                        <h1 className=''>Customers</h1>

```

Comment:
h1 está sem className. se for o proposito nao ter, pode remover o atributo className